modules/floating-buttons/classes/render/floating-buttons-render-base.php

```diff
@@ -381,10 +386,10 @@ protected function get_formatted_link( array $link, string $prefix ): string {
 				$formatted_link = ! empty( $link['username'] ) ? 'skype:' . $link['username'] . '?chat' : '';
 				break;
 			case Social_Network_Provider::WAZE:
-				$formatted_link = ! empty( $link['location'] ) ? $link['location'] : '';
+				$formatted_link = ! empty( $link['location'] ) ? $link['location']['url'] : '';
 				break;
 			case Social_Network_Provider::URL:
-				$formatted_link = ! empty( $link['url'] ) ? $link['url'] : '';
+				$formatted_link = ! empty( $link['url'] ) ? $link['url']['url'] : '';
```

Comment:
@mserino this looks wrong

Reply:
@nicoladj77 thanks for catching this. This code is actually unused anymore, I have replaced this with the proper link attributes now. Removed.